c10/build.bzl

```diff
@@ -1,6 +1,6 @@
 def define_targets(rules):
```

Comment:
Does Skylark supports optional arguments? If that is the case, I would suggest to use `name="c10"` argument and redefine it for internal builds